### PR TITLE
Update the comparison sheets

### PR DESCRIPTION
**Pull request by peci1**
I've reorganized the comparison sheet to express what SDF features are currently supported. I added info about missing closed kinematic chain support and a few more things.

### CODE REVIEW

**Comment by peci1** [comment]
Please, rather check all the checkmarks/crosses, they're just my best guess.

Also, I saw a checkmark near "Custom sensors" in Gazebo-classic (and Ignition Gazebo, too). What is meant by that? AFAIK the SDF spec doesn't allow any sensors except those explicitly mentioned in the `<sensor type>` attribute.

**Comment by peci1** [comment]
It's also highly probable that there are more SDF features I haven't identified which would be worth mentioning here.

File: blueprint/comparison.md [hunk 1]
```diff
@@ -151,6 +181,7 @@ ODE engine | ✓ |
 Bullet engine | ✓ |
 DART engine | ✓ | ✓ Plugin shipped with ign-physics
 Simbody engine | ✓ |
+TPE engine | ✕ | ✓
```
Comment (chapulina) [inline blueprint/comparison.md:184]: TPE is not on Blueprint

```suggestion
TPE engine | ✕ | ✕ (available in Citadel)
```

File: blueprint/comparison.md [hunk 2]
```diff
@@ -34,6 +35,27 @@ Custom update rate | ✓ | ✓
...
+## SDF Features
+
+Feature | Gazebo-classic | Ignition Gazebo
+-- | -- | --
+SDF frame semantics |✓| ✕ (available in Citadel)
+Load models from local files | ✓ | [✕](https://github.com/ignitionrobotics/ign-gazebo/issues/123)
+Closed kinematic chains | ✓  | [✕](https://github.com/ignitionrobotics/ign-physics/issues/25)
+Nested models | ✓ | Partial support
+Populations | ✓ | 
+Actors | ✓ |
```
Comment (chapulina) [inline blueprint/comparison.md:47]:
```suggestion
Actors | ✓ |  ✕ (available in Citadel)
```

File: blueprint/comparison.md [hunk 3]
```diff
@@ -34,6 +35,27 @@ Custom update rate | ✓ | ✓
...
+
+Feature | Gazebo-classic | Ignition Gazebo
+-- | -- | --
+SDF frame semantics |✓| ✕ (available in Citadel)
+Load models from local files | ✓ | [✕](https://github.com/ignitionrobotics/ign-gazebo/issues/123)
+Closed kinematic chains | ✓  | [✕](https://github.com/ignitionrobotics/ign-physics/issues/25)
+Nested models | ✓ | Partial support
+Populations | ✓ | 
+Actors | ✓ | 
+Markers | ✓ |
```
Comment (chapulina) [inline blueprint/comparison.md:48]:
```suggestion
Markers | ✓ |  ✕ (available in Citadel)
```

File: blueprint/comparison.md [hunk 4]
```diff
@@ -34,6 +35,27 @@ Custom update rate | ✓ | ✓
...
+Populations | ✓ | 
+Actors | ✓ | 
+Markers | ✓ | 
+Heightmaps | ✓ | [✕](https://github.com/ignitionrobotics/ign-gazebo/issues/237)
+DEM (Digital Elevation Models) | ✓ | [✕](https://github.com/ignitionrobotics/ign-gazebo/issues/235)
+Polylines | ✓ | [✕](https://github.com/ignitionrobotics/ign-gazebo/issues/186)
+World plugins | ✓ | ✓ Now called System plugin
+Model plugins | ✓ | ✓ Now called System plugin
+Sensor plugins | ✓ | ✓ Now called System plugin
+Visual plugins | ✓ | ✕ (available in Citadel)
```
Comment (chapulina) [inline blueprint/comparison.md:55]: Visual plugins as they exist on Gazebo-classic aren't really supported yet

```suggestion
Visual plugins | ✓ | [✕](https://github.com/ignitionrobotics/ign-gazebo/issues/265)
```

**Comment by chapulina** [comment]
>  I saw a checkmark near "Custom sensors" in Gazebo-classic (and Ignition Gazebo, too). What is meant by that?

Yeah that's a mistake, see https://github.com/ignitionrobotics/docs/pull/26/files#diff-9c8d54780a3947049a188bad343dabd2R39

**Comment by peci1** [comment]
Thanks for the corrections. I've accepted all of them.

How about the line with custom sensors that is fixed in #26? The PR doesn't seem like something that would get merged soon, but at least the correction of the line with custom sensors would deserve to be applied soon.

**Comment by chapulina** [comment]
> How about the line with custom sensors that is fixed in #26? The PR doesn't seem like something that would get merged soon

I'll put some time into that PR soon

**Pull request merged**